src/slots/InSlot.js

```diff
@@ -9,131 +8,32 @@ class InSlot extends Slot {
     return 'InSlot'
   }
 
-  static get DATA_TYPES() {
-    return {
-      INTEGER: 'integer',
-      FLOAT: 'float',
-      BOOLEAN: 'boolean',
-      ONE_OF: 'oneOf',
-    }
-  }
-
-  static SERIALIZE_OPTIONAL_DATA(inSlot) {
-    const optionals = {
-      min: inSlot.min,
-      max: inSlot.max,
-      selectOptions: inSlot.selectOptions,
-    }
-
-    return Object.entries(optionals).reduce((obj, [ key, value ]) => {
-      // eslint-disable-next-line no-param-reassign
-      if (value !== undefined) obj[key] = value
-      return obj
-    }, {})
-  }
-
   static get SCHEMA() {
     return Joi.object({
       type: Joi.string().allow(InSlot.TYPE).required(),
-      dataType: Joi.string().allow(...Object.values(InSlot.DATA_TYPES)).required(),
       tareable: Joi.boolean().required(),
```

Comment:
can all `InSlot`s be `tareable`?

Reply:
I was back and forth on this.  Probably only the integer and float. I'll change that (will require a couple of PRs on hambda to fix that)

Reply:
done